airflow/hooks/base_hook.py

```diff
@@ -96,3 +96,18 @@ def get_pandas_df(self, sql):
 
     def run(self, sql):
         raise NotImplementedError()
+
+    def run_with_advanced_retry(self, _retry_args, *args, **kwargs):
```

Comment:
I don't think we should add this in the `base_hook`, this should be just in the `http_hook`

Reply:
happy to move it back to http_hook